# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/base/CacheConstants.h
# Commit: 3cfe5bd5d084
# Full Hash: 3cfe5bd5d084baf18f5d95b0e879f6fc179fac34
# Author: Morgan Reschenberg <mreschenberg@mozilla.com>
# Date: 2021-10-06 04:09:04
# Regressor Bug: 1726227
# File Overlap Count: 1
# Description:
#   Bug 1726227: Cache parent-relative accessible bounds, resolution in parent process r=Jamie,eeejay
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D123399
# ==============================================================================

diff -r 76a965804210 -r 3cfe5bd5d084 accessible/base/CacheConstants.h
--- a/accessible/base/CacheConstants.h	Tue Oct 05 20:24:03 2021 +0000
+++ b/accessible/base/CacheConstants.h	Tue Oct 05 20:44:46 2021 +0000
@@ -15,6 +15,7 @@
   static constexpr uint64_t NameAndDescription = ((uint64_t)0x1) << 0;
   static constexpr uint64_t Value = ((uint64_t)0x1) << 1;
   static constexpr uint64_t Bounds = ((uint64_t)0x1) << 2;
+  static constexpr uint64_t Resolution = ((uint64_t)0x1) << 3;
   static constexpr uint64_t All = ~((uint64_t)0x0);
 };
 